### PR TITLE
[Merged by Bors] - Changed http:// to https:// on mailing list link

### PR DESCRIPTION
**Pull request by sandakersmann**
Changed http:// to https:// on mailing list link in README.md

### CODE REVIEW

**Review by michaelsproul — APPROVED** [review]
Thanks!

**Comment by realbigsean** [comment]
bors retry

File: README.md [hunk 1]
```diff
@@ -66,7 +66,7 @@ of the Lighthouse book.
 The best place for discussion is the [Lighthouse Discord
 server](https://discord.gg/cyAszAh).
 
-Sign up to the [Lighthouse Development Updates](http://eepurl.com/dh9Lvb) mailing list for email
+Sign up to the [Lighthouse Development Updates](https://eepurl.com/dh9Lvb) mailing list for email
```
Comment (michaelsproul) [inline README.md]:
```suggestion
Sign up to the [Lighthouse Development Updates](https://eepurl.com/dh9Lvb/) mailing list for email
```

This is a no-op change to re-trigger Github's knowledge of the base branch

**Comment by michaelsproul** [comment]
re-batching

bors r+

**Pull request closed without merging**